scripts/world_bank/health_nutrition_population/output/World_bank_hnp_population.tmcf

```diff
@@ -0,0 +1,9 @@
+Node: E: WorldBankPopulation ->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: C: WorldBankPopulation ->StatVar
+observationAbout: C:WorldBankPopulation ->Country
+observationDate: C:WorldBankPopulation->Year
+observationGender: C: WorldBankPopulation->Gender
+observationAge: C:WorldBankPopulation->Age
```

Comment:
Gender and Age can go into the statVar and don't need to be repeated in the statVarObservation.